pages/index.js

```diff
@@ -1,45 +1,12 @@
-import flatten from 'lodash/flatten';
+import React from 'react';
 
-import { initializeApollo } from 'lib/apolloClient';
-import { GET_CONTENT_FEED } from 'hooks/useContentFeed';
-import { GET_FEED_FEATURES } from 'hooks/useFeedFeatures';
-import { Layout } from 'components';
-
-import { FeedFeaturesProvider } from 'providers';
-import { HomeFeed } from 'components';
+import { FeatureFeedProvider } from '../providers';
+import { Layout, FeatureFeed } from '../components';
 
 export default function Home(props = {}) {
   return (
     <Layout title="Home">
-      <FeedFeaturesProvider Component={HomeFeed} />
+      <FeatureFeedProvider Component={FeatureFeed} />
     </Layout>
   );
-}
-
-export async function getServerSideProps() {
-  const apolloClient = initializeApollo();
-
-  const features = await apolloClient.query({ query: GET_FEED_FEATURES });
-  const data = features?.data?.userFeedFeatures;
-  const actions = flatten(data.map(({ actions }) => actions));
-  let promises = [];
-  actions.map(item =>
-    promises.push(
-      apolloClient.query({
-        query: GET_CONTENT_FEED,
-        variables: {
-          itemId: item?.relatedNode?.id,
-          child: true,
-          sibling: false,
-        },
-      })
-    )
-  );
-  await Promise.all(promises);
-
-  return {
-    props: {
-      initialApolloState: apolloClient.cache.extract(),
-    },
-  };
-}
```

Comment:
Is this removed because the query relies on `currentUser` and we can't server-side render it?

Reply:
Sorry about that I had removed it cause the queries that were being use had been kinda deprecated. But I re-added it with the new queries and it should be working now!